diff --git a/home/jon/Development/Personal/Teaching/php-tutorial-website/web/checkouts/rebase1/36008f07bb999835759399c2f513479dadf247f9/index.php b/home/jon/Development/Personal/Teaching/php-tutorial-website/web/checkouts/rebase1/29eab76549ccfb0594a9c5471b2353b0b806fa73/index.php
index 91ff8d1..21c29d7 100644
--- a/home/jon/Development/Personal/Teaching/php-tutorial-website/web/checkouts/rebase1/36008f07bb999835759399c2f513479dadf247f9/index.php
+++ b/home/jon/Development/Personal/Teaching/php-tutorial-website/web/checkouts/rebase1/29eab76549ccfb0594a9c5471b2353b0b806fa73/index.php
@@ -1,3 +1,26 @@
+<?php
+
+// Work out the path to the database, so SQLite/PDO can connect
+$root = __DIR__;
+$database = $root . '/data/data.sqlite';
+$dsn = 'sqlite:' . $database;
+
+// Connect to the database, run a query, handle errors
+$pdo = new PDO($dsn);
+$stmt = $pdo->query(
+	'SELECT
+		title, created_at, body
+	FROM
+		post
+	ORDER BY
+		created_at DESC'
+);
+if ($stmt === false)
+{
+	throw new Exception('There was a problem running this query');
+}
+
+?>
 <!DOCTYPE html>
 <html>
 	<head>
@@ -8,14 +31,20 @@
 		<h1>Blog title</h1>
 		<p>This paragraph summarises what the blog is about.</p>

-		<?php for ($postId = 1; $postId <= 3; $postId++): ?>
-			<h2>Article <?php echo $postId ?> title</h2>
-			<div>dd Mon YYYY</div>
-			<p>A paragraph summarising article <?php echo $postId ?>.</p>
+		<?php while ($row = $stmt->fetch(PDO::FETCH_ASSOC)): ?>
+			<h2>
+				<?php echo htmlspecialchars($row['title']) ?>
+			</h2>
+			<div>
+				<?php echo $row['created_at'] ?>
+			</div>
+			<p>
+				<?php echo htmlspecialchars($row['body']) ?>
+			</p>
 			<p>
 				<a href="#">Read more...</a>
 			</p>
-		<?php endfor ?>
+		<?php endwhile ?>

 	</body>
-</html>
+</html>